File: 프로그래머스/1/12917. 문자열 내림차순으로 배치하기/문자열 내림차순으로 배치하기.java
class Solution {
    public String solution(String s) {
        char [] ch = new char[s.length()];

        for(int i = 0; i < s.length(); i++){
            ch[i] = s.charAt(i);
        }
        char temp = 'a';
        for(int i = 0; i < s.length(); i++){
            for(int j = i+1; j < s.length(); j++){
                if(ch[i] < ch[j]){
                    temp = ch[j];
                    ch[j] = ch[i];
                    ch[i] = temp;
                }
            }
        }
        
        String answer = new String(ch);
        return answer;
    }
}

File: 프로그래머스/1/12917. 문자열 내림차순으로 배치하기/문자열 내림차순으로 배치하기.cs
public class Solution {
    public string solution(string s) {
        char [] ch = new char[s.Length];

        for(int i = 0; i < s.Length; i++){
            ch[i] = s[i];
        }
        char temp = 'a';
        for(int i = 0; i < s.Length; i++){
            for(int j = i+1; j < s.Length; j++){
                if(ch[i] < ch[j]){
                    temp = ch[j];
                    ch[j] = ch[i];
                    ch[i] = temp;
                }
            }
        }
        
        string answer = new string(ch);
        return answer;
    }
}